security/production-secure-deploy/internal-rolebindings/cfrb-internal-ksqldb-0.yaml

```diff
@@ -0,0 +1,18 @@
+# rolebinding `internal-ksqldb-0` gives user `ksql` the role 
+# `ResourceOwner` for the cluster named `ksql-cluster`
+apiVersion: platform.confluent.io/v1beta1
+kind: ConfluentRolebinding
+metadata:
+  name: internal-ksqldb-0
+spec:
+  # Cluster id is <namespace>.<ksql_cluster_name>_
+  clustersScopeByIds:
+    ksqlClusterId: confluent.ksqldb_ 
```

Comment:
It might be better to have namespace in metadata since we specify ksqlClusterId pattern with <namespace>.<ksql_cluster_name>_ here.